api/src/main/java/net/kyori/adventure/identity/Identity.java

```diff
@@ -30,13 +30,14 @@
 import org.checkerframework.checker.nullness.qual.NonNull;
 
 /**
- * An identity.
+ * An identity used to track the sender of messages for the social interaction features
+ * introduced in Minecraft: Java Edition 1.16.4+.
  *
  * @since 4.0.0
```

Comment:
Perhaps use `@sinceMinecraft 1.16.4` here?

Reply:
This feature has been supported since 1.16.0 -- it was only associated with game functionality since 1.16.4 though.